### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Sudarshan519**

### CODE REVIEW

File: lib/app/modules/reset_password/views/otpForm.dart [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                  obscureText: true,
+                  obscuringCharacter: '*',
+                  style: otptextStyle,
+                  keyboardType: TextInputType.number,
+                  textAlign: TextAlign.center,
+                  decoration: InputDecoration(border: InputBorder.none),
+                  onChanged: (value) => nextField(value, pin4FocusNode),
+                ),
+              ),
+              Container(
```
Comment (imsujan276) [inline lib/app/modules/reset_password/views/otpForm.dart]: separate it into widget

**Review by imsujan276 — CHANGES_REQUESTED** [review]

File: lib/app/modules/menupage/views/menupage_view.dart [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+          Obx(
+        () => controller.currentIndex != 0
+            ? Container()
+            : IndexedStack(
+                index: controller.currentIndex,
+                children: _children,
+              ),
+      ),
+
+      bottomNavigationBar: Obx(
```
Comment (imsujan276) [inline lib/app/modules/menupage/views/menupage_view.dart]: move it to separate widget

File: lib/app/modules/menupage/views/menupage_view.dart [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                        flex: 1,
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+      ),
+
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
+      floatingActionButton: Obx(
```
Comment (imsujan276) [inline lib/app/modules/menupage/views/menupage_view.dart]: move it to separate widget

File: lib/app/modules/menupage/views/menupage_view.dart [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+          onPressed: () {
+            controller.updateCurrentIndex(0);
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class NavItem extends StatelessWidget {
```
Comment (imsujan276) [inline lib/app/modules/menupage/views/menupage_view.dart:150]: move widgets into widgets folder

File: lib/app/modules/onboarding/views/onboarding_view.dart [hunk 5]
```diff
@@ -15,60 +16,100 @@ class OnboardingView extends GetView<OnboardingController> {
...
+        children: [
+          Container(
+            child: PageView.builder(
+                controller: controller.pageController,
+                onPageChanged: (i) {
+                  controller.onChanged(i);
+                },
+                itemCount: splashData.length,
+                itemBuilder: (_, i) {
+                  if (context.isPortrait)
```
Comment (imsujan276) [inline lib/app/modules/onboarding/views/onboarding_view.dart]: separate the conditional widgets

**Review by imsujan276 — CHANGES_REQUESTED** [review]

File: lib/app/modules/more/bindings/more_binding.dart [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import 'package:get/get.dart';
+
+import '../controllers/more_controller.dart';
+
+class MoreBinding extends Bindings {
```
Comment (imsujan276) [inline lib/app/modules/more/bindings/more_binding.dart:5]: use appropriate and meaningful page/module name.

**Review by imsujan276 — CHANGES_REQUESTED** [review]

File: lib/app/modules/reset_password/views/otpForm.dart (hunk 1 above)
Comment (Sudarshan519) [reply lib/app/modules/reset_password/views/otpForm.dart]: done

File: lib/app/modules/menupage/views/menupage_view.dart (hunk 2 above)
Comment (Sudarshan519) [reply lib/app/modules/menupage/views/menupage_view.dart]: done

File: lib/app/modules/menupage/views/menupage_view.dart (hunk 3 above)
Comment (Sudarshan519) [reply lib/app/modules/menupage/views/menupage_view.dart]: done

File: lib/app/modules/menupage/views/menupage_view.dart (hunk 4 above)
Comment (Sudarshan519) [reply lib/app/modules/menupage/views/menupage_view.dart:150]: done

File: lib/app/modules/more/bindings/more_binding.dart (hunk 6 above)
Comment (Sudarshan519) [reply lib/app/modules/more/bindings/more_binding.dart:5]: done

File: lib/app/modules/onboarding/views/onboarding_view.dart (hunk 5 above)
Comment (Sudarshan519) [reply lib/app/modules/onboarding/views/onboarding_view.dart]: done

**Pull request merged**